src/content/ja/ai-capabilities/frame-interpolation.md

```
@@ -5,9 +5,71 @@ section: ai-capabilities
+昔のカクカクした動画を滑らかにしたり、スローモーションで下がったfpsを補ったりする用途で、かなり昔から使われています。
+
+また、動画生成AIの登場によって、**ジェネレーティブフレーム補間** という、単なるFPS補間以上の技術も生まれています。
+
+---
+
+## fpsを上げるためのフレーム補間（古典的VFI）
+
+一般的なVFIは、時間的に近い2枚のフレーム（0.1秒未満程度）を受け取り、その間に挟まる「中間フレーム」を1枚以上生成します。これを繰り返すことで、動画全体のフレーム数を増やします。
+
+
+![](https://gyazo.com/af7273352979b5286d8f85a9b6915ab6.png){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)